src/routes/acquirerRoutes.ts

```diff
@@ -384,6 +383,40 @@ router.post("/:resourceID/:step", async (req: Request, res: Response) => {
     formdata.stepHistory = [];
   }
 
+  if (req.body.addCountry) {
+    console.log("stepData", stepData);
```

Comment:
Thanks Amir, there are a couple leftover `console.logs` in the acquirerRoutes.ts file please can you remove them 🙂

Reply:
Nice catch, thanks Maddie.